style.css

```diff
@@ -326,6 +326,13 @@
     height: 48px;
     color: #fff;
     cursor: pointer;
```

Comment:
- [ ] Hello team you did an excellent job on the project 📈,but your report should list the result of the evaluation you did. PLease remember to add it on the description of your pull request. ♻️

![image](https://user-images.githubusercontent.com/95243996/193791325-d9575fab-440f-47b9-b3d7-85ac60c2a8cb.png)

Reply:
Hi @andgarzonmal thanks for the feedback